test/e2e/credentials/setup.go

```diff
@@ -55,11 +62,46 @@ func Setup(ctx context.Context, logger logr.Logger, awsSession *session.Session,
 		return nil, err
 	}
 
+	instances, err := ec2Client.DescribeInstances(ctx, &ec2.DescribeInstancesInput{
+		Filters: []types.Filter{
+			{
+				Name:   aws.String("tag:" + constants.TestClusterTagKey),
+				Values: []string{*cluster.Cluster.Name},
+			},
+			{
+				Name:   aws.String("tag:Jumpbox"),
+				Values: []string{"true"},
+			},
+			{
+				Name:   aws.String("instance-state-name"),
+				Values: []string{"running"},
+			},
+		},
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	keypair, err := ec2Client.DescribeKeyPairs(ctx, &ec2.DescribeKeyPairsInput{
```

Comment:
This is really the only reason we need the keypair, to get the public key. The key pair is never used in an AWS sense, attached to a node.  Its somewhat handy to "store" it in aws, but we could instead just read the pub key from the local file and/or config.